sdk/python/kfp/_local_client.py

```diff
@@ -343,14 +351,16 @@ def _generate_cmd_for_docker_execution(
         pipeline: dsl.Pipeline,
         op: dsl.ContainerOp,
         stack: Dict[str, Any],
+        docker_options: List[str] = []
     ) -> List[str]:
         """Generate the command to run the op in docker locally."""
         cmd = self._generate_cmd_for_subprocess_execution(
             run_name, pipeline, op, stack)
-
+        
```

Comment:
nit: remove extra whitespace. 

PS: you can use yapy to format the code. Ref: https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#sdk